scripts/setup/requirements.python_tests.txt

```diff
@@ -0,0 +1,8 @@
+# scripts/tests
+click
+colorama
+
+# src/python_testing
+mobly
+pyasn1
+pyasn1_modules
```

Comment:
The individual tests also have some requirements - ex DA tests require cryptography. Would the idea be to just add dependent packages here for the individual tests?

Reply:
- We should have all important packages by default
- We should not manage packages by test (too error prone)
- PLEASE let's not add more deps. We should always try to do without a new dep. The number of existing deps is already huge.

Reply:
I believe cryptography and others will be added as a sideffect for now and it would work. Ideally we should have as few dependencies as possible and explain when we add new ones why they are needed as they seem to have reasonably large overhead (even if it is just waiting for them to be installed).

For now I would say lets keep the list that works and if we ever get problems, we should update the list (or re-assess the need for a dependency in the first place).